### PR TITLE
Longer KO time from blunt, only skull and neck crits paralyze.

### PR DESCRIPTION
**Pull request by saltprjak**
## About The Pull Request

Title. Now only skull (zone at the top of the head) and neck blunt crits can disable you. Paralysis-inducing fractures need more incurred damage to have a chance to happen, other fractures became a little easier. Now possible to fracture someone's eye and jaw.
Ear and eye fractures aren't paralysis-inducing, but can instakill crit-weak enemies.
Embed chance on fracture crits didn't work, so removed it to prevent misleading text (and clubs "embedding" into eyes with no effect). Made the text more catch-all.
Reduces fracture wounds' HP, meaning they are easier to heal.
Paralyzing fractures require more prior damage to trigger.
Unconcsiousness time from head-crits upped to thirty seconds, minute and thirty seconds from behind.

## Why It's Good For The Game

More control over which people you're trying to vegetable. Knockouts now act more than a very short stun - most of the times they run out quicker than you can do anything with your victim, leading to situations such as guards having to break someone's spine in order to get them to stop wrenching their feet off.
Less paralysis gaming overall, since it's usually a very unfun situation to end up in - both for the victim, and the people that have to take care of and entertain them until a surgeon or a cleric can be found.

Head fractures still incur a ton of pain and, coupled with damage needed to reach them, will almost always send someone into paincrit after a few seconds. the various fractures on the head also stack, yummers

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by ThePainkiller** [comment]
This PR:
- Reduces fracture wounds' HP, meaning they are easier to heal, **and this is not mentioned on the PR**
- Makes it so that targetting eyes and mouth can cause a knockdown, which is a literal nerf to their wounds as that makes you flip a coin between the actual lethal critwound (please don't do this)
- Makes certain fractures (depressed cranial fracture, mandibular fracture, and neck dislocation) harder to achieve, necessitating more damage to reach them, **also not mentioned**
- Takes away the lethality of eye fracture, **also not mentioned by the PR** (denied btw, keep that in)

Please detail **everything** changed on the PR body. The embeds issue is fine, but this would make maces even less lethal than they already are.

**Comment by Cre77** [comment]
> This PR:
> 
> * Reduces fracture wounds' HP, meaning they are easier to heal, **and this is not mentioned on the PR**
> * Makes it so that targetting eyes and mouth can cause a knockdown, which is a literal nerf to their wounds as that makes you flip a coin between the actual lethal critwound (please don't do this)
> * Makes certain fractures (depressed cranial fracture, mandibular fracture, and neck dislocation) harder to achieve, necessitating more damage to reach them, **also not mentioned**
> * Takes away the lethality of eye fracture, **also not mentioned by the PR** (denied btw, keep that in)
> 
> Please detail **everything** changed on the PR body. The embeds issue is fine, but this would make maces even less lethal than they already are.

crap, i forgot to mention those. Thanks. As far as I've tested - the knockdowns still occur. Mouth and jaw crits don't crit-kill weak mobs, but they still stack a LOT of pain with fractures. While it's true that TTK with  blades is overall better, neither of these changes touches either knockdowns, or crits - you can still disable people as normal by targetting specific parts of the head, and you can still knock them down and knock them out. There is "lethal"-lethal blunt attacking on anything that isn't crit-weak.

Lethality of the eye fracture was assumed more from the proximity of the brain, and the general survivability of a fracture there compared to one directly to the cranium - I'll put that back in.

**Comment by Cre77** [comment]
To add to the upper point, the tresholds for fractures (especially those with perma-disabling) were raised SPECIFICALLY to give people more chances to knock someone out - seeing as KOs are arguably deadlier, while also allowing apprehending people without inflicting permanent, crippling damage. I'd need some more specific info on that.
(also, yeah, i looked it up - a punch to the nose is about the least likely one to knock someone out, compared to directly head, or jawline. hence why it's not a knockout zone.)

File: code/datums/wounds/fractures.dm [hunk 1]
```diff
@@ -56,10 +56,10 @@
 		"The skull caves in!",
 	)
 	sound_effect = "headcrush"
-	whp = 150
+	whp = 60
```
Comment (ThePainkiller) [inline code/datums/wounds/fractures.dm:59]:
```suggestion
	whp = 80
```

**Review by ThePainkiller — DISMISSED** [review]

**Comment by ThePainkiller** [comment]
Might give this one a testmerge to see if people are happier with maces being slightly better at their knockout job
I might do something in the future to make sure certain wounds sustained by maces can be lethal, perhaps execution-style.
Make maces great again.

**Comment by NPC1314** [comment]
Time to test this baby then

**Comment by Coastfront** [comment]
For feedback; This feels really nice to use.

I've gotten to play a few-rounds with the TM for this, & Having an actually-fairly consistent way to knock people unconscious & play nonlethally has both been greatly-enjoyable, & Has made blunt-weapons feel-less like swords that kill slower.

**Comment by NPC1314** [comment]
No bugs been reported, noticed a blunt weapon runtime in the logs but probably unrelated, going to study it a little later today and if not related this PR getting fullmerged today

**Review by ThePainkiller — APPROVED** [review]
Tested, can't replicate the runtime. After good feedback and overal toying around, this is good for a full merge.

**Pull request merged**